Gemfile

```diff
@@ -20,7 +19,7 @@ gem 'uglifier', '>= 1.3.0'
 gem 'coffee-rails', '~> 4.2'
 # See https://github.com/rails/execjs#readme for more supported runtimes
 # gem 'therubyracer', platforms: :ruby
-
+gem 'rubocop', '~> 0.51.0', require: false
 # Use jquery as the JavaScript library
```

Comment:
We dont need rubocop in production.
Move to development group

Reply:
moved